### DISCUSSION TITLE
【求助】如何在苹果手机上使用绳网？

### DISCUSSION THREAD

**Discussion by tyhpoi** [share121/inter-knot#1109] [其他]
![pexels-nitin-dhumal-46710](https://github.com/user-attachments/assets/bfecfb2c-a746-4855-a002-5ac1cea3ef0c)

现在只能在电脑上用绳网，苹果手机我能安装绳网助手但是网络不行一直处于加载状态。{{{(>_<)}}}

**Comment by 13855422937cyc** [comment]
我电脑用不了手机可以

**Comment by GT025233** [comment]
这得问传奇绳匠法厄同了。有答案了踢我一下
